torch/nn/attention/_score_mod.py

```diff
@@ -0,0 +1,64 @@
+"""This module defines the common used score_mod functions for templated attention in PyTorch."""
+import torch
+
+
+def identity(
```

Comment:
I think add these to the 'templated_attention' file

Reply:
also maybe keep em underscored for now so you dont have to worry about docs

Reply:
SG!